lib/-private/todo-handler.js

```diff
@@ -55,7 +55,13 @@ module.exports = class TodoHandler {
       }
     }
 
+    const today = new Date();
+
     [...existingTodos.values()].forEach((todo) => {
+      if (todo.errorDate instanceof Date && today > todo.errorDate) {
```

Comment:
Should we normalize these dates somehow? I _think_ `new Date()` is going to be for the current date and time (including timezone?).

It would be bad/odd if things pass in CI because it runs in GMT but locally I'm running in EST/PST...

Reply:
Ya, we should be setting the hour/min/second to 0 I believe.

Reply:
seems good, but I think that will still have the GMT / PDT issue won't it?

Reply:
I think this means we want to set `createdDate` to that in addition, in todo-utils.

Reply:
Either way, I think if we encapsulate the logic in the utils we don't have to perfectly nail this one

Reply:
Yep, agree. Will update that repo.

Reply:
I think we need to store the dates in UTC, and do the local conversion and comparison from that. Timezone comparisons are particularly finicky, so I'll see what I can come up with.

Reply:
Done.